src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -100,12 +100,16 @@ public function getProperties($class, array $context = array())
      */
     public function getTypes($class, $property, array $context = array())
     {
-        if ($fromMutator = $this->extractFromMutator($class, $property)) {
```

Comment:
Is is really necessary to change this (it will make it harder to merge bug fixes later)?

Reply:
Nope, it was just done in order to optimize the code. I can revert it if you want

Reply:
let's revert yes please :)

Reply:
It's reverted